src/App.jsx

```diff
@@ -1,9 +1,70 @@
-import './App.css';
+import { Route, Routes, Link } from "react-router-dom";
+import "./App.css";
+import ContactsListPage from "./ContactsListPage";
+import ContactDetailsPage from "./ContactDetailsPage";
+import CreateNewContactFormPage from "./CreateNewContactFormPage";
+import { useEffect, useState } from "react";
 
 function App() {
-    return (
-        <p>Hello, world!</p>
-    );
+  const [contactInfoList, setConstactInfoList] = useState([
+    {
```

Comment:
Why not just use: const [contactInfoList, setContactInfoList] = useState([]);

Reply:
I wanted a specific set of information from the API, when expanding to the extensions of the exercise I will make that change. To be able to take in all data and display it nicely.